src/test/java/com/wikia/webdriver/testcases/mcfootertests/NonEnAnonMixedContentFooterTests.java

```diff
@@ -0,0 +1,84 @@
+package com.wikia.webdriver.testcases.mcfootertests;
+
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.helpers.User;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.mcfooter.MixedContentFooter;
+
+import org.testng.annotations.Test;
+
+@Test(groups = {"NonEnAnonMixedContentFooter"})
+@Execute(onWikia = "de.gta", asUser = User.ANONYMOUS)
+
+public class NonEnAnonMixedContentFooterTests extends NewTestTemplate{
+
+  @Test
+  public void isMCFooterPresentOnNonENwiki() {
+    MixedContentFooter MCFooter = new MixedContentFooter(driver);
+    MCFooter.openWikiMainPage().scrollToFooter();
+    Assertion.assertTrue(MCFooter.isMcfooterPresent());
+  }
+
+  @Test
+  public void exploreWikisCardIsPresentOnNonENwiki(){
+    MixedContentFooter MCFooter = new MixedContentFooter(driver);
+    MCFooter.openWikiMainPage().scrollToFooter();
+    Assertion.assertTrue(MCFooter.isExploreWikisCardPresent());
+  }
+
+  @Test
+  public void discussionsCardIsPresentOnNonENwiki(){
+    MixedContentFooter MCFooter = new MixedContentFooter(driver);
+    MCFooter.openWikiMainPage().scrollToFooter();
+    Assertion.assertTrue(MCFooter.isDiscussionsCardPresent());
+  }
+
+  @Test
+  @Execute(onWikia = "es.serowiec")
+  public void discussionsCardIsPresentOnNonENwikiWithEmptyDiscussions(){
+    MixedContentFooter MCFooter = new MixedContentFooter(driver);
+    MCFooter.openWikiMainPage().scrollToFooter();
+    Assertion.assertTrue(MCFooter.isDiscussionsCardPresent());
+  }
+
+  @Test
+  @Execute(onWikia = "es.gta")
+  public void discussionsCardIsNotPresentOnNonENwikiWithoutDiscussions(){
+    MixedContentFooter MCFooter = new MixedContentFooter(driver);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS00117)